Source/missiles.cpp

```diff
@@ -1538,10 +1538,14 @@ void AddRuneExplosion(Missile &missile, const AddMissileParameter & /*parameter*
 		dmg = ScaleSpellEffect(dmg, missile._mispllvl);
 
 		missile._midam = dmg;
+		
+		constexpr Displacement Offsets[] = { Displacement(Direction::North), Displacement(Direction::NorthWest), { 0, 0 }, 
+											 Displacement(Direction::SouthEast), Displacement(Direction::West), Displacement(Direction::SouthWest), 
+											 Displacement(Direction::South), Displacement(Direction::East), Displacement(Direction::NorthEast)};

```

Comment:
Maybe to keep this simpler we could introduce Direction::None, also I suggest putting each on a separate line:

```suggestion
		constexpr Direction Offsets[] = {
			Direction::North,
			Direction::NorthWest,
			Direction::None,
			Direction::SouthEast,
			Direction::West,
			Direction::SouthWest,
			Direction::South,
			Direction::East,
			Direction::NorthEast,
		};
```